web/src/scripts/containers/ProjectNavigator.jsx

```diff
@@ -104,17 +108,20 @@ class ProjectNavigator extends Component {
 
   handlePreviewClick = async (node) => {
     const {rootPath, textEditorCompositeActions} = this.props
+    this.setState({currentPreview: node.path})
```

Comment:
Yep, planning to do that in the next PR. Wasn't sure if this was the final interaction or anything, and the redux store for this stuff is still in motion. Will improve it.